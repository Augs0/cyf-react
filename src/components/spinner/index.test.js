import React from 'react';
import { shallow } from 'enzyme';

import Spinner from './';

describe('Spinner Component', () => {
  it('renders correctly', () => {
    expect(shallow(<Spinner />)).toMatchSnapshot();
  });
});
