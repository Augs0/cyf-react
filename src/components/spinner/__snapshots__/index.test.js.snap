// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component renders correctly 1`] = `
<Spinner
  className="css-1slz332"
  fadeIn="full"
  name="circle"
  noFadeIn={false}
  overrideSpinnerClassName=""
/>
`;
